packages/lib-classifier/src/plugins/drawingTools/models/marks/Mark/Mark.js

```diff
@@ -89,24 +89,26 @@ const BaseMark = types.model('BaseMark', {
       self.finished = true
     }
 
-    function setSubTaskVisibility (visible, drawingMarkNode, previousAnnotationValues) {
+    function setPreviousAnnotations(previousAnnotationValues) {
+      if (previousAnnotationValues?.length > 0) {
+        previousAnnotationValues.forEach((previousAnnotationValue) => {
+          self.subTaskPreviousAnnotationValues.put(previousAnnotationValue)
+        })
+      } else {
+        self.subTaskPreviousAnnotationValues.clear()
+      }
+    }
+
+    function setSubTaskVisibility (visible, markBounds) {
```

Comment:
Small naming quibble here, I'd use `visibility` rather than `visible` to  more clearly indicate it should be a boolean. Non-blocking suggestion as that was what it was previously.

Reply:
Good catch.